### PR TITLE
docs(radio): comsummate radio group options type

### PR DESCRIPTION
**Pull request by winchesHe**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

image

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Comsummate RadioGroup options type     |
| 🇨🇳 Chinese |      完善 RadioGroup 的 options 类型     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/radio/index.zh-CN.md [hunk 1]
```diff
@@ -39,30 +39,43 @@ demo:
...
+| disabled     | 禁选所有子单选器                                       | boolean                                                                       | false     |       |
+| name         | RadioGroup 下所有 `input[type="radio"]` 的 `name` 属性 | string                                                                        | -         |       |
+| options      | 以配置形式设置子元素                                   | string\[] \| number\[] \| Array&lt;[CheckboxOptionType](#CheckboxOptionType)> | -         |       |
+| optionType   | 用于设置 Radio `options` 类型                          | `default` \| `button`                                                         | `default` | 4.4.0 |
+| size         | 大小，只对按钮样式生效                                 | `large` \| `middle` \| `small`                                                | -         |       |
+| value        | 用于设置当前选中的值                                   | any                                                                           | -         |       |
+| onChange     | 选项变化时的回调函数                                   | function(e:Event)                                                             | -         |       |
```
Comment (afc163) [inline components/radio/index.zh-CN.md:65]: 不要列等宽，这样在窄屏幕看上去很丑。

Comment (winchesHe) [reply components/radio/index.zh-CN.md]: 收到，确实好丑哈哈哈，给它改回去了🙌🙌

File: components/radio/index.zh-CN.md [hunk 2]
```diff
@@ -58,12 +58,25 @@ demo:
...
 | size | 大小，只对按钮样式生效 | `large` \| `middle` \| `small` | - |  |  |
 | value | 用于设置当前选中的值 | any | - |  |  |
 | onChange | 选项变化时的回调函数 | function(e:Event) | - |  |  |
 
+### CheckboxOptionType
+
+| Property | Description | Type | Default | Version |
+| --- | --- | --- | --- | --- |
+| label | 用于作为 Radio 选项展示的文本 | `string` | - | 4.4.0 |
+| value | 关联 Radio 选项的值 | `string` &#124; `number` &#124; `boolean` | - | 4.4.0 |
```
Comment (li-jia-nan) [inline components/radio/index.zh-CN.md:72]: 这个用转义符复杂化了吧，感觉没必要，用反斜杠专义就好了，可以参考其它地方

Comment (winchesHe) [reply components/radio/index.zh-CN.md]: Fixed

**Comment by li-jia-nan** [comment]
CI 挂了，需要更新一下快照

**Review by afc163 — APPROVED** [review]

**Comment by li-jia-nan** [comment]
- #48313

**Pull request merged**